program/c/makefile

```diff
@@ -3,3 +3,6 @@ SOLANA := ../../../solana
 include $(SOLANA)/sdk/bpf/c/bpf.mk
 cpyth:
 	bash -c "ar rcs target/libcpyth.a target/**/*.o"
+cpythtest:
+	gcc -c ./src/oracle/for_cargo_test/cpyth_test.c -o ./target/cpyth_test.o
```

Comment:
I feel bad about using this "naked" `gcc` for some reason.

Reply:
I think you can use $CC